codeql-workspace.yml

```diff
@@ -1,2 +1,2 @@
 provide:
-  - "*/codeql-packs/**/qlpack.yml"
\ No newline at end of file
+  - "tutorial-queries/codeql-pack.yml"
```

Comment:
Cool! If that removes the lines, that's so much simpler than manually sending update events to VSCode. 

>  In future, if we generate new subfolders (like codeql-custom-queries-xxx), we might need to add those paths to the codeql-workspace.yml file.

Should we also add a wildcard here for skeleton folders so they don't get red underlines? 

```
- "codeql-custom-queries-*/codeql-pack.yml"
```

Reply:
> Should we also add a wildcard here for skeleton folders so they don't get red underlines?

This isn't necessary until we add those as _subfolders_ in the workspace (at the moment they get added as top-level workspace folders).

Can we do this as part of https://github.com/github/codespaces-codeql/pull/25, since that's the PR that changes the skeleton pack behaviour?